runelite-client/src/main/java/net/runelite/client/plugins/gargs/GargsPlugin.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2018, https://runelitepl.us
+ * Copyright (c) 2018, https://github.com/runeliteplusplus
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.gargs;
+
+import net.runelite.client.eventbus.Subscribe;
+import java.util.Iterator;
+import java.util.ArrayList;
+import net.runelite.api.events.GameTick;
+import net.runelite.client.plugins.PluginType;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.api.NPC;
+import javax.annotation.Nullable;
+import net.runelite.client.ui.overlay.OverlayManager;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.Plugin;
+
```

Comment:
This should be added in the Grotesque Guardian plugin instead of adding a new plugin for the same activity with possibly duplicate features.